database/seeders/SupplierSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Supplier;
+use Illuminate\Database\Seeder;
+
+class SupplierSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     * itemsテーブルに対して1つしか紐づかない前提
+     * @return void
+     */
+    public function run()
+    {
+        Supplier::factory(10)->create([
+            'user_id' => 1,
```

Comment:
SupplierFactoryでは、user_id => 1 と指定されてるのでこの記述は不要かと思います。
```suggestion
        Supplier::factory(10)->create();
```